jobwatch/skawatch.py

```diff
@@ -74,8 +74,10 @@ def __init__(self, task, maxage=1, dbfile=None, table=None, timekey='tstart'):
 perl_errs = set(('uninitialized value',
                  '(?<!Program caused arithmetic )error',
                  'warn', 'fatal', 'fail', 'undefined value'))
-arc_exclude_errors = [r'warning:\s+\d+\s',
-                      'file contains 0 lines that start with AVERAGE']
+arc_exclude_errors = [
+    r'warning:\s+\d+\s',
+    'file contains 0 lines that start with AVERAGE',
+    'p3 fluence not plotted, error : cannot do a non-empty take from an empty axes.']
```

Comment:
I'm know I'm missing the context, but at first glance this sure *looks* like an error that should not be ignored.

Reply:
Sure.  I just thought we were doing a fine job of ignoring it now so could perhaps continue.

Reply:
Not paying attention at the moment is slightly different from permanently ignoring.

Reply:
Sure.  But hard to figure out which things need to be red at the skawatch level, and which things are OK for verbose output in the email watch logs level, and which things we're just always going to ignore.  Maybe for skawatch we don't care at *all* about checking the arc logs and just want to know all the files were updated sometime recently.

In general, I would like to see more review of warnings and filters in jobwatch as a natural part of making changes to trending applications.